client/templates/index.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <meta name="fragment" content="!">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

Comment:
@impactmass I included that line because I noticed a distortion in style on iphone 5.  Not sure about the downsides though. @zenweasel @rymorgan what do you think?